File: breeding-mice/breeding.js
function fibonacci(number) {
    var sqRootOf5 = Math.sqrt(5);

    var Phi = (1+sqRootOf5)/2;
    var phi = (1-sqRootOf5)/2

    return Math.round((Math.pow(Phi, number) - Math.pow(phi, number)) / sqRootOf5);
}

let N = 6;
console.time("execution time:");
for (let i =0; i < 18; i++){
    console.log(fibonacci(N))
}
console.timeEnd("execution time:");

File: breeding-mice/breeding.java
public class breeding {
    public static void main(String[] args){
        long startTime = System.nanoTime(); 

        //start of simulation
        int N = 6; 

        for (int i = 0; i < 18; i++){
            System.out.println("Total pairs: " + breedingSimulator(i));
        }

        long endTime = System.nanoTime();

        //end of simulation

        System.out.println("Execution time in ns: " + (endTime - startTime));
    }

    public static int breedingSimulator(int N) {
        int pair = (int) ((Math.pow(((1 + Math.sqrt(5))/2), N) - Math.pow(-((1 + Math.sqrt(5))/2), -N))/Math.sqrt(5));
        return pair;
    }
}
